src/FNAPlatform/SDL3_FNAPlatform.cs

```diff
@@ -2017,9 +2017,19 @@ private static void INTERNAL_AddInstance(uint dev)
 
 			// Open the device!
 			INTERNAL_devices[which] = SDL.SDL_OpenGamepad(dev);
+			string error = SDL.SDL_GetError();
+			if (!string.IsNullOrEmpty(error)) {
```

Comment:
Similar to the other checks we should use tue return value of the function instead of the error buffer